docs/destinations/forwarding-events/google-tag-manager-server-side/snowplow-client-for-gtm-ss/snowplow-client-configuration/index.md

```diff
@@ -48,6 +48,36 @@ If using this Client to receive Snowplow Tracker events and then forward to a Sn
 
 By default, the self describing event will be "shredded" into a key using the schema name as the key, this is a "lossy" transformation, as the Minor and Patch parts of the jsonschema version will be dropped. This flag populates the original, lossless, Self Describing Event as `x-sp-self_describing_event`.
 
+Let's assume we have a self-describing event following the schema `iglu:com.acme/foobar/jsonschema/1-0-0`. By default, the option to *Include Original Self Describing Event* is disabled. So the Snowplow client by default will include it in the common event as:
+
+```
+'x-sp-self_describing_event_com_acme_foobar_1': {
+    foo: 'bar'
+}
+```
```

Comment:
Actually, is this supposed to be JSON? If so:
````suggestion
```json
"x-sp-self_describing_event_com_acme_foobar_1": {
    "foo": "bar"
}
```
````